collectfast/boto.py

```diff
@@ -1,8 +1,21 @@
 from . import settings
 
+try:
+    from storages.backends.s3boto import S3BotoStorage
+    from storages.backends.s3boto3 import S3Boto3Storage
+    has_boto = True
+except:
+    has_boto = False
+
 
 def is_boto3(storage):
-    return type(storage).__name__ == 'S3Boto3Storage'
+    return has_boto and isinstance(storage, S3Boto3Storage)
```

Comment:
would this work if we didn't have boto/boto3 installed?

Reply:
Yeah, the `isinstance(storage, S3Boto3Storage)` will only get evaluated if `has_boto` is `True`, which will only happen if the import is successful. Would be good to test this but I think it might be a lot of effort.